mikado-app/src/viewmodel/displayLayerStore.js

```diff
@@ -178,28 +178,54 @@ class DisplayLayerOperations {
     });
   }
 
+  modifyNodePosition(position, nodeDim, viewport) {
+    
+    // hacky code that checks every position starting from the specified pos
+    // and ending when a pos is found or there is no possible position in current
+    // viewport
+    
+    const { nodes } = this.#state;
+    for (let i = position.y; i < viewport.y - nodeDim.height; i+=30) {
+      for (let j = position.x; j < viewport.x - nodeDim.width; j+=92) {
```

Comment:
In the issue, it was supposed to keep trying to add. If it went off screen, it would scroll there.

But deferring to new issue.